patches/minecraft/net/minecraft/entity/monster/EntityCreeper.java.patch

```diff
@@ -0,0 +1,30 @@
+--- ../src-base/minecraft/net/minecraft/entity/monster/EntityCreeper.java
++++ ../src-work/minecraft/net/minecraft/entity/monster/EntityCreeper.java
+@@ -184,10 +184,13 @@
+ 
+     public void func_70645_a(DamageSource p_70645_1_)
+     {
++        if (net.minecraftforge.common.ForgeHooks.onLivingDeath(this, p_70645_1_)) return;
```

Comment:
it'd be better, and probably simpler to just fix this weird vanilla implementation.
Move the spawning of the items below to a override of the dropEquipment function. And let our normal handlers deal with it.